patches/chrome-browser-resources-side_panel-read_anything-BUILD.gn.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/resources/side_panel/read_anything/BUILD.gn b/chrome/browser/resources/side_panel/read_anything/BUILD.gn
+index 22549e7de5e3a77c9eb8b0c4a34f54e3fe6a5808..888f315ddf2005e665affd3b89f21d942430af42 100644
+--- a/chrome/browser/resources/side_panel/read_anything/BUILD.gn
++++ b/chrome/browser/resources/side_panel/read_anything/BUILD.gn
+@@ -48,7 +48,7 @@ build_webui("build") {
+             target_gen_dir) ]
+ 
+   webui_context_type = "untrusted"
+-  optimize = optimize_webui
++  optimize = false
```

Comment:
is there a follow-up to this?

Reply:
Filed https://github.com/brave/brave-browser/issues/38422.

Reply:
Well, I meant a follow-up for the chromium update ideally. Not optimizing this doesn't seem great cc @petemill

Reply:
We don't even use this WebUI, so not sure we need to invest time finding a fix for the cause yet?